EasyPost.Integration/cassettes/net/synchronous/async.json

```diff
@@ -0,0 +1,101 @@
+[
```

Comment:
thought: can we get these to be treated as binary like the other cassettes so they aren't expanded in reviews?

Reply:
Resolved